BlackRevival.APIServer/Controllers/NoticeController.cs

```diff
@@ -35,7 +39,11 @@ public IActionResult GetGacha(string id, string lang)
         var queryString = HttpContext.Request.QueryString.Value;
         _logger.LogInformation("Query string: {QueryString}", queryString);
 
-        string json = System.IO.File.ReadAllText("Data/Config/GachaNotices.json");
+		if (!System.IO.File.Exists("Data/Config/GachaNotices.json"))
+		{
+			return NotFound();
+		}
+		string json = System.IO.File.ReadAllText("Data/Config/GachaNotices.json");
```

Comment:
If possible, I would make those magic strings static somewhere in the top of the file